pkg/scheduler/util.go

```diff
@@ -51,6 +51,8 @@ func loadSchedulerConf(confStr string) ([]framework.Action, []conf.Tier, error)
 	if err := yaml.Unmarshal(buf, schedulerConf); err != nil {
 		return nil, nil, err
 	}
+	framework.ApplyPluginConfDefaultFuncs(schedulerConf)
+
```

Comment:
just set default value of plugins one by one here, or provide a defaultFunc here.